tests/test_client.py

```diff
@@ -60,7 +60,7 @@ def test_client_authentication_url(api_client, api_url):
             ('state', ''),
             ('redirect_uri', '/redirect'),
             ('response_type', 'code'),
-            ('client_id', 'None'),
+            ('client_id', ''),
```

Comment:
Hmm, this is technically a breaking change. I know it's sorta weird, but would it be possible to go back to the old behavior and change the earlier line in `authentication_url` to `'client_id': self.app_id or 'None',`?